windows/src/desktop/help/index.md

```diff
@@ -2,16 +2,51 @@
 title: Keyman 14.0 Help
 ---
 
-* [About Keyman](about/)
+Need help using Keyman for Windows? You'll find everything you need here, including product documentation,
+frequently asked questions, tutorials, and videos.
 
-* [Getting Started](start/)
+## [Getting Started](start/tutorial)
+* [Step-by-step Guide](start/tutorial)
+* [Installing a keyboard](start/download-and-install-keyboard)
 
-* [Basic Help](basic/)
+## [Common Questions](common/)
+* [Which operating systems does Keyman for Windows work with?](common/requirements)
+* [Does Keyman for Windows run on Mac?](common/mac)
 
-* [Advanced Help](advanced/)
+## Ask for Help
+[Ask other users in the SIL Keyman Keyman Community](https://community.software.sil.org/c/keyman)
 
-* [Common Questions](common/)
+## Help Topics
 
-* [Troubleshooting](troubleshooting/)
+### [Troubleshooting](troubleshooting/)
+* [Square boxes or fonts not showing correctly](https://help.keyman.com/kb/42)
+* [How to send a diagnostic report to us](https://help.keyman.com/kb/40)
+* [Right-to-left text out of order](https://help.keyman.com/kb/41)
+* [Windows and Word language configuration](https://help.keyman.com/kb/63)
 
-* [Contextual Help](context/)
+### [Using Keyman for Windows](basic/)
+* [Configuration Tasks](basic/config/)
+* [Language Switcher](basic/language-switcher)
+* [On Screen Keyboard](basic/toolbox/osk)
+* [Font Helper](basic/toolbox/font-helper)
+
+### [Advanced Topics](advanced/)
+* [Character Map](basic/toolbox/character-map)
+* [Hotkeys](start/hotkey_set)
+* [Text Services Framework Addin](advanced/text_services_framework)
+* [Use Word macros to switch keyboard and font](https://help.keyman.com/kb/33)
+* [Application Programming Interface (API)](https://help.keyman.com/developer/engine/desktop/current-version/api/)
+* [Keyman for Windows version history](https://help.keyman.com/products/windows/version-history)
```

Comment:
This won't be live to production just yet

Reply:
That's probably okay.